src/services/users/signin-user-by-email.test.js

```diff
@@ -0,0 +1,52 @@
+const { expect } = require('chai')
+const db = require('../../db')
+const {fixtures} = require('../../../test/utils')
+const signInUserByEmail = require('./signin-user-by-email')
+
+describe('User signin', () => {
+	let user
+	let validPassword
+	beforeEach(async () => {
+		user = await fixtures.insertUser({
+			password : 'validPaswword123$'
+		})
+		validPassword = 'validPaswword123$'
+	})
+
+	it('should throw an error if user does not exist', async () => {
+		const { id,email } = user
+		await db.query(
+			`DELETE FROM users
+             WHERE id = $1`, [id])
+
+		return expect(signInUserByEmail(email , validPassword))
+			.to.be.rejectedWith(
+				'Invalid email or password.')
+	})
+
+	it('should signin user', async () => {
```

Comment:
So technically there's nothing to sign in into, description should be about what you want to see. maybe returns right response or something

Reply:
okay thanks